tests/benchmark/mergesort/c/mergesort.c

```diff
@@ -44,7 +44,7 @@ static bool sorted(int *tab, int size) {
     return true;
 }
 
-#define N 2000000
+#define N 100000
```

Comment:
Changing this number to a smaller one significantly changes the character of the benchmark. With 100k elements in the list one needs at most a few 64k pages, so memory management / allocation impact is no longer tested here.

Reply:
I'll change it back once the pr is ready. It is very useful for debugging purposes to be able to run the benchmarks faster